src/schema/rules/files/deriv/atlas.yaml

```
@@ -0,0 +1,64 @@
+---
+volumetric:
+  suffixes:
+    - dseg
+    - probseg
+    - mask
+  extensions:
+    - .nii
+    - .nii.gz
+    - .tsv
+    - .json
+  datatypes: []
+  entities:
+    atlas: required
+    space: optional
+    resolution: optional
+    description: optional
+
+surface:
+  suffixes:
+    - dseg
+    - probseg
+    - mask
+  extensions:
+    - .label.gii
```

```suggestion
     - .gii
     - .label.gii
```
To account for scalar-based atlases